packages/core/demo/demo-data/bar.ts

```diff
@@ -52,10 +52,14 @@ export const groupedBarData = {
 
 export const groupedBarOptions = {
 	scales: {
+		headers:{
+			title:  "2018 Annual Sales Figures"
+		},
```

Comment:
`title` is really an option for the entire chart. doesn't have anything to do with the scales.

I would move `title` to the root layer of `groupedBarOptions`